index.rst

```diff
@@ -437,284 +329,111 @@ this case, we may have something like:
-*nearly* identical attributes; available labels vs selected label, available
-mapping vs selected map, available branch description vs. selected branch
-description and so on.
+    #.  Rename ``recommendedSettingsLabels`` to ``labels``.
+    #.  Rename ``recommendedSettingsVersion`` to ``versions``.
+    #.  Rename ``settingsUrl`` to ``url``.
+    #.  Add ``mapping``.
+        For a configuration repository ``mapping`` will reflect the content of the ``_labels.yaml`` file.
+        For a configuration database ``mapping`` will explicitly show the mapping of ``label`` to ``label:version``.
 
-.. _section-appendix-renaming:
+#. Rename ``settingsApplied`` to ``configurationApplied``
 
-Topic renaming
---------------
+    #.  Add ``label``.
+    #.  Add ``version``.
+    #.  Add ``mapping``.
+    #.  Add ``url``.
+    #.  Rename ``otherSettingsEvent`` -> ``otherInfo``.
 
-The interface would be much clear if we rename some topics and attributes to
-reflect more closely their true meaning.
+  The event will publish the selected values once the CSC is configured.
 
-The following is a renaming suggestion for discussion:
+#. In the ``start`` command, rename attribute ``settingsToApply`` to ``configuration``.
+
+.. _section-continuous-monitoring:
+
+Continuous monitoring configuration repository
+----------------------------------------------
+
+Right now CSCs are required to publish ``configurationsAvailable``  (former ``settingsVersions``, see :ref:`renaming proposal <section-renaming>`) when they transition to ``STANDBY`` state.
+Nevertheless, while in ``STANDBY`` state it is possible for someone to update the available configuration, which would make the information out of sync.
+We propose that, while in ``STANDBY`` state, CSCs continuously monitor the configuration repository and update the information if needed.
+If an invalid configuration is made available while the CSC is in ``STANDBY`` the CSC should transition to ``FAULT`` state.
+
+.. _section-default-configuration:
+
+Base configuration (handling default configuration values)
+----------------------------------------------------------
+
+This is mainly a proposal to update how Salobj manages default configuration values.
+Other :ref:`handcrafted CSCs <section-handcrafted>` are encouraged to follow this proposal as closely as possible to maintain uniformity across the system.
 
-    #.  Rename ``settingsVersions``to``configurationsAvailable``
+As described :ref:`above <section-salobj>`, CSCs written with Salobj define a configuration schema (e.g. `ts_atdome <https://github.com/lsst-ts/ts_ATDome/blob/develop/schema/ATDome.yaml>`__).
+The configuration schema contains default values for the configuration which are loaded if the ``start`` command is sent with an empty ``configuration`` attribute (the default value).
+Nevertheless, the values in the schema are seldom valid beyond a unit testing environment, which requires users to provide some kind of *operational defaults* or *default label*.
```

Comment:
I'd add something about how this is causing more confusion than benefit (if you want to say why the current system is weak).

Reply:
Ok, I took a crack at it... let me know what you think....

Reply:
Looks fine. Small grammar issue "has two different meaningS"